src/basho_stats_utils.erl

```diff
@@ -61,9 +61,9 @@ r_port() ->
     try port_command(Port, "write('', file=stdout())\n") of
         _ ->
             receive
-                {Port, {data, {eol, []}}} ->
```

Comment:
This change feels suspicious to me. From the commit comment, it looks like you are just trying to fix a dialyzer warning, but it seems like this actually changes the functionality, right? Before the change, it would have only matched this receive clause if it was from the same port that we got back from `open_port` above, but now it'll match anything in that variable.

Maybe that's fine in practice? I don't have time right now to figure that out for sure, but wanted to check with you and see if this is something you had already considered.